fe/fe-core/src/main/java/com/starrocks/connector/iceberg/cost/IcebergStatisticProvider.java

```diff
@@ -95,6 +99,10 @@ private IcebergFileStats generateIcebergFileStats(IcebergTable icebergTable, Sca
                 if (icebergFileStats == null) {
                     icebergFileStats = new IcebergFileStats(dataFile.recordCount());
                 } else {
+                    if (files.contains(dataFile.path().toString())) {
```

Comment:
why not use  datafile  start = 0 to check wether this datafile  have been used?

Reply:
Good idea, will do it later.